src/main/java/com/aws/greengrass/componentmanager/ComponentStore.java

```diff
@@ -506,7 +516,8 @@ public RecipeMetadata getRecipeMetadata(ComponentIdentifier componentIdentifier)
 
             throw new PackageLoadingException(String.format(
                     "Failed to get recipe metadata because the recipe metadata file json has wrong structure."
```

Comment:
do we need to distinguish _every_ type of invalid JSON?

Reply:
cleaned up a bit